.prettierignore

```diff
@@ -13,4 +13,4 @@ pnpm-lock.yaml
 public/staticwebapp.config.json
 
 # Generated files
-tailwind.config.ts
+*.generated.*
```

Comment:
I wonder if this isn't working as expected

Reply:
why? did you confuse this with gitignore?

Reply:
yup. totally confused it with gitignore